library/src/androidTest/java/org/xmtp/android/library/GroupTest.kt

```diff
@@ -342,4 +342,41 @@ class GroupTest {
             assertEquals(conversation.topic, awaitItem().topic)
         }
     }
+
+    @Test
+    fun testCanAllowGroup() {
+        val group = boClient.conversations.newGroup(
+            listOf(
+                alix.walletAddress,
+                caro.walletAddress
+            )
+        )
+
+        var result = boClient.contacts.isGroupAllowed(group.id)
+
+        assert(!result)
```

Comment:
I assumed that this test should fail now since the default is allow on newGroup?